support-frontend/assets/components/socialShare/socialShare.jsx

```diff
@@ -24,29 +24,30 @@ type SocialMedia = {
   a11yHint: string,
 };
 
+type WindowFeatures = '' | 'menubar=no, toolbar=no, resizable=yes, scrollbars=yes, width=500, height=400';
 
 // ----- Setup ----- //
 
 const socialMedia: {
   [Platform]: SocialMedia,
```

Comment:
I feel that `SharePlatform` could be a better name since it includes email?

Reply:
Fixed

Reply:
actually or `PlatformParams` to correspond to the `Platform` key